File: Flutter App/unicoVehicle/lib/Models/ShowroomReview.dart
import './Miscellaneous/Showroom.dart';
import './Miscellaneous/User.dart';

class ShowroomReview {
  final int? showroomReviewId;
  final Showroom? showroom;
  final User? user;
  final String? review;

  ShowroomReview({
    this.review,
    this.showroom,
    this.showroomReviewId,
    this.user,
  });
}

File: UnicoVehicle/UnicoVehicle/UnicoVehicle.DTO/ShowroomReview.cs
using System;
namespace UnicoVehicle.DTO
{
    public class ShowroomReview
    {
        public int ShowroomReviewId { get; set; }
        public Miscellaneous.Showroom Showroom { get; set; }
        public Miscellaneous.User User { get; set; }
        public string Review { get; set; }
    }
}
